src/client/components/AddResume/AddResume.js

```diff
@@ -1,47 +1,60 @@
-import React, {useState} from 'react';
+import React, { useState } from 'react';
 import ReactDom from 'react-dom';
+import {useHistory} from 'react-router-dom';
+
 import Button from '../Button/Button';
 import './AddResume.css';
 import close from '../../assets/images/closeIcon.svg';
 import { useStorage } from '../../hooks/fileUploader';
+import { useAuthentication } from '../../hooks/useAuthentication';
 
 export const AddResume = () => {
   const [title, setTitle] = useState('');
   const [description, setDescription] = useState('');
   const [error, setError] = useState(null);
-  const [uploadedFile, setUploadedFile] = useState('');
+  const { uploadedFile, setUploadedFile, uploadFile, uploadToStorage, url } = useStorage();
+  const { userData } = useAuthentication();
+  const history = useHistory();
+
+  // console.log(url);
+  // console.log(uploadedFile)
+  
 
   const types = [
     'application/pdf',
     'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
     'application/msword',
     'text/plain',
-    'text/rtf'
+    'text/rtf',
   ];
 
   const handleChange = (e) => {
     const selectedFile = e.target.files[0];
-
     setUploadedFile(selectedFile);
 
     if (selectedFile) {
       if (types.includes(selectedFile.type)) {
         setError(null);
         setUploadedFile(selectedFile);
```

Comment:
I think we can remove setUploadedFile and uploaded file?

Reply:
we only can remove setUploadedFile inside the condition. if we remove them altogetherthen we are missing the hook which upload file into firebase.